test/unit/lib/urlutilTest.js

```diff
@@ -86,8 +86,11 @@ describe('urlutil', function () {
       it('is localhost (case-insensitive)', function () {
         assert.equal(UrlUtil.isNotURL('LoCaLhOsT'), false)
       })
-      it('ends with period (input contains a forward slash)', function () {
-        assert.equal(UrlUtil.isNotURL('https://brave.com/test/cc?_ri_=3vv-8-e.'), false)
+      it('ends with period (input contains a forward slash and domain)', function () {
+        assert.equal(UrlUtil.isNotURL('brave.com/test/cc?_ri_=3vv-8-e.'), false)
+      })
+      it('ends with period (input contains only a forward slash)', function () {
+        assert.equal(UrlUtil.isNotURL('brave/com/test/cc?_ri_=3vv-8-e.'), true)
```

Comment:
Testing for an extra .might not be a good idea, because folks may use Brave to browse their intranet. It's not a best practice, but many folks will setup a host file entry and access hosts by computer name (instead of domain)